src/pages/CommitDetailPage/CommitBundleAnalysis/CommitBundleAnalysis.spec.tsx

```diff
@@ -105,6 +105,7 @@ const mockRepoOverview = ({
   coverageEnabled = false,
 }) => ({
   owner: {
+    isCurrentUserActivated: true,
```

Comment:
Dumb question - what does it mean for a user to be "activated"? Is that the same `me.onboardingCompleted`?

Reply:
Not dumb at all! activated simply means they own a seat and are authorized to view private repos in the organization

Reply:
Got it, thank you!!